### PR TITLE
Updated file with new strings

### PR DESCRIPTION
**Pull request by Joachiem**
Updated the file with all new strings.

Discord: Joachiem#2966

### CODE REVIEW

File: nl.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
-    "name" : "Nederlands",
```
Comment (Krymonota) [inline nl.json]: Why did you remove that? It was correct.

File: nl.json [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
-    "name-en" : "Dutch",
```
Comment (Krymonota) [inline nl.json]: Why did you remove that? It was correct.

File: nl.json [hunk 3]
```diff
@@ -1,6 +1,5 @@
...
+    "name" : "Dutch",
```
Comment (Krymonota) [inline nl.json]: `name` should be the name of the language in this language. However, this change is not necessary. See above.

File: nl.json [hunk 4]
```diff
@@ -47,7 +46,7 @@
             },
             "translationNotAvailable" : "Helaas is er geen vertaling beschikbaar voor deze pagina.",
             "translationIncomplete" : "Niet alle tekst op deze pagina zal worden vertaald."
-        },
+        {,
```
Comment (Krymonota) [inline nl.json]: That was correct before. Your change breaks the whole file, so it can't be used as a translation.

File: nl.json [hunk 5]
```diff
@@ -21,7 +20,7 @@
                 "December"
             ],
             "types" : {
-                "backpack" : "Rugzak",
+               "backpack" : "Rugzak",
```
Comment (Krymonota) [inline nl.json]: The spacing was correct before. Please don't remove the space before the translation key.

File: nl.json [hunk 6]
```diff
@@ -61,9 +60,9 @@
                 "reminders" : "herinneringen",
                 "history" : "winkel geschiedenis"
             },
-            "news" : "nieuws",
+              "news" : "nieuws",
```
Comment (Krymonota) [inline nl.json]: The spacing was correct before. Please don't add unnecessary spaces before the translation key.

File: nl.json [hunk 7]
```diff
@@ -72,15 +71,21 @@
                 "discord" : "Discord",
                 "twitter" : "Twitter"
             },
-            "api" : "api",
+           "api" : "api",
```
Comment (Krymonota) [inline nl.json]: The spacing was correct before. Please don't remove the space before the translation key.

File: nl.json [hunk 8]
```diff
@@ -90,7 +95,7 @@
             "error-too-long" : "Dit duurt langer dan gedacht"
         },
         "shop" : {
-            "title" : "Huidige winkel",
+           "title" : "Huidige winkel",
```
Comment (Krymonota) [inline nl.json]: The spacing was correct before. Please don't remove the space before the translation key.

File: nl.json [hunk 9]
```diff
@@ -130,6 +138,7 @@
                 "line1" : "Dit is de item shop rotatie van [date] voor Fortnite Battle Royale. ",
                 "line2" : "Klik op een item om meer informati te zien.",
                 "today-button" : "Item Shop van vandaag"
+
```
Comment (Krymonota) [inline nl.json]: Newline? Not needed.

File: nl.json [hunk 10]
```diff
@@ -139,54 +148,126 @@
             "more-ingame-button" : "Meer Laden"
         },
         "randomiser" : {
-            "title" : "Willekeuriger",
+           "title" : "Willekeurige items",
```
Comment (Krymonota) [inline nl.json]: The spacing was correct before. Please don't remove the space before the translation key.

File: nl.json [hunk 11]
```diff
@@ -139,54 +148,126 @@
...
-                "title" : "Uh oh!",
-                "line1" : "Er is iets misgegaan tijdens het verwerken. Probeer het later opnieuw!",
-                "line2" : "Je kan [hier](/oauth) opnieuw proberen in te loggen."
+                "title" : "Oh nee!",
+                "line1" : "Er is iets fout gegaan tijden het verwerken, probeer het later opnieuw!",
+                "line2" : "Je kan proberen om opnieuw in te loggen, klik dan [hier](/oauth)."
             }
         }
     }
-}
```
Comment (Krymonota) [inline nl.json]: Why did you remove that? It was correct before. Your change breaks the whole file, so it can't be used as a translation.

File: nl.json [hunk 12]
```diff
@@ -37,7 +37,7 @@
                 "wrap" : "Wikkel",
                 "bundle" : "Bundel",
                 "misc" : "Overig"
-            },
+             },
```
Comment (Krymonota) [inline nl.json]: One space too much! :P

File: nl.json (hunk 10 above)
Comment (Krymonota) [reply nl.json]: Oops, probably my fault! Of course there has to be a space before the translation key.

File: nl.json [hunk 13]
```diff
@@ -139,54 +148,127 @@
             "more-ingame-button" : "Meer Laden"
         },
         "randomiser" : {
-            "title" : "Willekeuriger",
+           "title" : "Willekeurige items",
```
Comment (Krymonota) [inline nl.json]: Oops, probably my fault! Of course there has to be a space before the translation key.

**Comment by Krymonota** [comment]
Perfect! @Joachiem 

Looks good now to be merged.

**Comment by Joachiem** [comment]
👍
